src/docs/pages/guides/testing.md

```diff
@@ -0,0 +1,104 @@
+---
+eleventyNavigation:
+  key: Testing
+  parent: Guides
+  order: 10
+layout: layout.njk
+title: Testing
+permalink: guides/testing/
+---
+
+# Testing
+
+If you want to __unit test__ code in your app, which depends on the functionality of Liquid Oxygen Web Components, without mocking their internals, then you must make sure all relevant components have _hydrated_ in your test setup, before you execute your test code. You can do so by running the `defineCustomElements()` method imported from `'@emdgroup-liquid/liquid/dist/loader'` before each test and a `await`ing a helper function, which makes sure all components involved in the test have hydrated.
```

Comment:
```suggestion
If you run __unit tests__ in your application, which depend on the functionality of Liquid Oxygen Web Components, you must ensure that the components have been _hydrated_. 

Run the following steps to prepare your tests:  
1. Run `defineCustomElements()` imported from `'@emdgroup-liquid/liquid/dist/loader'` before running the tests 
2. In your test, `await` all components to be hydrated. We created a helper function below, which you can use.
```

Reply:
I think the part about mocking components is important. A developer could decide to not hydrate the Web Components, and mock them instead, in order to reduce code execution to, and to have more control over the actual test case. So, I would keep this part. But I like the list structure.

Reply:
Definitely get the point. Hmmm.
Still think we should not overload the introduction. How about considering `mocking` as an advanced topic. Either as separate paragraph of the intro or as a separate passage after the example?

Maybe something like:
> Alternative to hydration you can also mock Liquid Oxygen Components in your tests. 🤷‍♂️

Reply:
I have changed it to:

```md
If you run __unit tests__ in your application, which depend on the functionality of Liquid Oxygen Web Components, and you do not want to mock that functionality, you must ensure that the components have been _hydrated_ before you execute your test code:

1. Call `defineCustomElements()` imported from `'@emdgroup-liquid/liquid/dist/loader'` before each test.
2. `await` all components to be hydrated. You can use a helper function similar to the one in the example test code below.
```

Doesn't look too complex to me.

Reply:
I think that the option to mock the functionality is so important, that it should be mentioned early on.

Reply:
Alright. I like this one!